tests/integration/test_manylinux.py

```diff
@@ -175,7 +175,7 @@ def tmp_docker_image(base, commands, setup_env={}):
 
     logger.info("Made image %s based on %s", image.short_id, base)
     try:
-        yield image.short_id
+        yield image.id
     finally:
```

Comment:
I left more details in the commit description, but this makes the tests work under Podman and seems relatively benign. Happy to revert if you prefer.